MapleServer2/Types/Stats.cs

```diff
@@ -0,0 +1,545 @@
+﻿using Maple2Storage.Types;
+using Maple2Storage.Types.Metadata;
+
+namespace MapleServer2.Types;
+
+// Player Stats in Packet Order - Count: 35 (0x23)
+public enum StatId : byte
```

Comment:
Delete this and use Maple2Storage.Enums.ItemAttribute? maybe rename the enum too